examples/Getting Started with Semiconductor Module/Getting Started with Semiconductor Module.py

```diff
@@ -0,0 +1,215 @@
+""" Getting Started with Semiconductor Module example.
+
+    Example NI TSM Test Program using simulated NI-Digital and NI-DCPower instruments
+"""
+
+import random
+
+import nidcpower
+import nidigital
+from nitsm.codemoduleapi import (
+    SemiconductorModuleContext,
+    code_module,
+    Capability,
+    InstrumentTypeIdConstants,
+)
+
+
+@code_module
+def open_dcpower_sessions(tsm_context: SemiconductorModuleContext):
+    """Open NI-DCPower instrument sessions."""
+    resource_strings = tsm_context.get_all_nidcpower_resource_strings()
+    for resource_string in resource_strings:
+        session = nidcpower.Session(
+            resource_string,
+            options={"Simulate": True, "DriverSetup": {"Model": "4143", "BoardType": "PXIe"}},
+        )
+        tsm_context.set_nidcpower_session(resource_string, session)
+
+
+@code_module
+def open_digital_sessions(tsm_context: SemiconductorModuleContext):
+    """Open NI-Digital instrument sessions."""
+    instrument_names = tsm_context.get_all_nidigital_instrument_names()
+    for instrument_name in instrument_names:
+        session = nidigital.Session(
+            instrument_name, options={"Simulate": True, "driver_setup": {"Model": "6570"}}
+        )
+        tsm_context.set_nidigital_session(instrument_name, session)
+
+
+@code_module
+def open_all_instruments_sessions(tsm_context: SemiconductorModuleContext):
+    """Open all instrument sessions."""
+    open_dcpower_sessions(tsm_context)
+    open_digital_sessions(tsm_context)
+
+
+@code_module
+def continuity(
+    tsm_context: SemiconductorModuleContext,
+    pins,
+):
+    """Measure continuity on configured Pins (NI-DCPower, NI-Digital).
+
+    This example uses simulated data and driver calls so it can run on a system without the
+    NI-Digital Pattern or NI-DCPower drivers installed. Refer to the Accelerometer example
```

Comment:
Not entirely accurate. Still need the drivers installed to simulate the instruments.  Lets change to:
```suggestion
    This example uses simulated data and driver calls so it can run on a system without the
    NI-Digital Pattern or NI-DCPower instruments installed. Refer to the Accelerometer example
```